src/lib.rs

```diff
@@ -898,6 +1130,7 @@ impl<'a> RenderPass<'a> {
         );
     }
 
+    /// Draws primitives using the bound index buffer.
```

Comment:
note: this is written as an alternative to "draw" that draws from vertex buffers, but in fact it's also using vertex buffers. We need to adjust the docs to reflect that and avoid confusion.

Reply:
see https://github.com/gfx-rs/wgpu-rs/pull/11/commits/96e8784098076e6491cb8179e6fd01f51a62574f#diff-b4aea3e418ccdb71239b96952d9cddb6R1135